app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Vocable</string>
+
+    <!-- Presets -->
+    <string name="select_something">Wähle unten etwas aus, um zu sprechen</string>
+    <string name="phrases_page_number">Seite %1d von %2d</string>
+    <string name="my_sayings_empty"><b>Ihre Redewendungen werden hier angezeigt.\n</b>\nNavigieren Sie zu der Tastatur, geben Sie etwas ein und\n wählen Sie dann den \"☆\", um es zu speichern.</string>
+    <string name="my_sayings_empty_handset_port"><b>Ihre Redewendungen werden hier angezeigt.\n</b>\nNavigieren Sie zu der Tastatur, geben Sie etwas ein und\n wählen Sie dann den \"☆\", um es zu speichern.</string>
+    <string name="my_sayings_empty_handset_land"><b>Ihre Redewendungen werden hier angezeigt.\n</b>\nNavigieren Sie zu der Tastatur, geben Sie etwas ein und wählen Sie dann den \"☆\", um es zu speichern.</string>
+    <string name="my_sayings_empty_settings"><b>Sie haben noch keine Redewendungen gespeichert. Wählen Sie das "+" oben rechts, um etwas hinzuzufügen.</b></string>
+
+    <!-- Settings Page -->
+    <string name="settings">Einstellungen</string>
+    <string name="privacy_policy">Datenschutz</string>
+    <string name="contact_developers">Entwickler kontaktieren</string>
+    <string name="version">V %s</string>
```

Comment:
Doesn't have to do with this PR (so nbd), but should try and prefer [Android string formatting](https://developer.android.com/guide/topics/resources/string-resource#formatting-strings) over Java formatting

Reply:
ah i see okay, thank you!